src/entity.ts

```diff
@@ -67,7 +67,7 @@ export class Entity<T> {
             }
         }
 
-        context.done(null, returnState);
+        return returnState;

```

Comment:
I'm leaning towards this _would_ be breaking. Since we have an upcoming breaking change release soon (for the new programming model), I'd prefer to postpone fixing until then.

I have not been able to get an entity function to work locally yet, but I believe previously you could ignore the result of the function and it would still work because of this `context.done`. Like this:
```typescript
const myEntityFunc = df.entity(function* (context) {
    // do stuff
});

export default (context) => {
    myEntityFunc(context); // note no `return` here
};
```

I don't know why you'd wrap `myEntityFunc` like that, but doesn't really matter from a breaking change perspective.

Reply:
Yes you're right. I am able to run entities locally, and I was able to verify that the pattern you had above does work normally after this change but will stop working after it. Is there a way around that while still not using `context.done()` or are we okay with breaking this scenario?

P.S. not sure if that's why you couldn't get entities to work locally for you, but `df.entity()` takes a normal function as an argument not a generator like `df.orchestrator()`

Reply:
>  Is there a way around that while still not using context.done() or are we okay with breaking this scenario?

I don't think there is a way without `context.done()`. And no, we should not do breaking changes to existing functionality without a major version release. It's worth noting that this bug is only about cleaning up the logs - the existing error doesn't actually block the function from working.

> P.S. not sure if that's why you couldn't get entities to work locally for you, but df.entity() takes a normal function as an argument not a generator like df.orchestrator():

Yes that was my problem, thanks!